userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -148,6 +148,24 @@ guessSyntax: true
 {{< /tab >}}
 {{< /tabpane >}}
 
+If you are using a Docsy version later than `0.6.0`, the code blocks show a
```

Comment:
```suggestion
If you are using a Docsy version `0.6.0` or later, code blocks show a
```

Reply:
This does not work for `0.6.0` and we haven't definitely decided on the next version number. I think "later than" covers that. About the article "the", although I know  I don't really need it here, I try to never omit it, based on this guideline: https://developers.google.com/style/articles